pennylane/math/multi_dispatch.py

```diff
@@ -152,6 +153,14 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+@multi_dispatch(argnum=[0])
+def kron(*args, like=None, **kwargs):
+    """The kronecker/tensor product of args."""
+    if like == "scipy":
+        return onp.kron(*args, **kwargs)  # Dispatch scipy kron to numpy backed specifically.
+    return np.kron(*args, **kwargs)
+
+

```

Comment:
This was required to avoid a deprecation warning as `scipy.kron` will be deprecated in the next cycle !